examples/clientApplication/client/index.js

```diff
@@ -3,8 +3,8 @@ $(document).ready(function() {
         var setCount = 0;
         setCount += prePopulateValues(e, 'configure-login-service-dialog-clientId', 'clientApplication');
         setCount += prePopulateValues(e, 'configure-login-service-dialog-secret', '12345');
-        setCount += prePopulateValues(e, 'configure-login-service-dialog-baseUrl', 'http://localhost:3100');
```

Comment:
While this appears counter intuitive, the examples/clientApplication example is intended to work with the examples/resourceServer which runs on localhost:3100.